init.lua

```diff
@@ -787,7 +786,7 @@ require('lazy').setup({
       -- ... and there is more!
       --  Check out: https://github.com/echasnovski/mini.nvim
     end,
-  },
+  },docker
```

Comment:
The added `docker` here looks like a mistake and causes stylua to complain and also running nvim with this config gives an error:
```
Error detected while processing .../init.lua:
E5113: Error while calling lua chunk: .../init.lua:791:
  attempt to call global 'docker' (a nil value)
```

Reply:
Cripes, that must've been an accidental paste or such.
Fixed!